src/app/homepage/home-page.component.html

```diff
@@ -12,86 +11,89 @@ <h1><b>Welcome to Summer of Good Deeds! </b></h1>
-          <div class="w3-row">
+          <div class="w3-container">
+            <p>You can choose activities that seem meaningful and important to you. Let us carefully consider who needs
+              our
+              time, strength, advice, help, understanding. Let's go to the nearest communities, our native cities, let's
+              talk to
+              nearby organizations, neighborhoods, hard-working families. Maybe they need help?</p>
+            <div class="w3-row">
+            </div>
           </div>
         </div>
-      </div>
 
-    </div>
+      </div>
 
 
-    <div class="w3-col l4">
-
-      <div class="w3-card w3-margin w3-margin-top">
-        <div class="w3-container w3-white">
-          <h4><b>When to volunteer?</b></h4>
-          <ul>
-            <li>In 2019, you can register for the project and do a good job in May-October.</li>
-            <li>Each employee has one free and paid working day for good work.</li>
-          </ul>
-          <p><b>What support is available?</b><br> Each employee receives an amount of EUR 15, which can be used to perform
-            a good
-            job (eg purchase the necessary tools, etc.).</p>
-          <p><b>What's next?</b><br> After a good job, the team captain is invited to share the impressions and photos on
-            the blog
-            of this site</p>
-          <p>Our communality and good will is what can lead to change. Let's do a good job together!</p>
+      <div class="w3-col l4">
+
+        <div class="w3-card w3-margin w3-margin-top">
+          <div class="w3-container w3-white">
+            <h4><b>When to volunteer?</b></h4>
+            <ul>
+              <li>In 2019, you can register for the project and do a good job in May-October.</li>
+              <li>Each employee has one free and paid working day for good work.</li>
+            </ul>
+            <p><b>What support is available?</b><br> Each employee receives an amount of EUR 15, which can be used to
+              perform
+              a good
+              job (eg purchase the necessary tools, etc.).</p>
+            <p><b>What's next?</b><br> After a good job, the team captain is invited to share the impressions and photos
+              on
+              the blog
+              of this site</p>
+            <p>Our communality and good will is what can lead to change. Let's do a good job together!</p>
+          </div>
         </div>
-      </div>
-      <hr>
+        <hr>
 
 
+      </div>
+
     </div>
+    <br>
 
   </div>
-  <br>
-
-</div>
 </div>
-<!--<footer class="w3-container w3-dark-grey w3-padding-32 w3-margin-top">-->
-  <!--<p><a href="https://www.w3schools.com/w3css/default.asp" target="_blank"></a></p>-->
-<!--</footer>-->
+<footer class="w3-container w3-dark-grey w3-padding-32 w3-margin-top">
+  <p><a href="https://www.w3schools.com/w3css/default.asp" target="_blank"></a></p>
```

Comment:
why do you need a link to w3school tutorial

Reply:
Čia net nežinau ar man tvarkyt, ar ne, nes čia nuo kito stable barancho rebasinta ir liko šita nesąmonė